devices/Framework.js

```diff
@@ -41,6 +41,9 @@ export const FrameworkSingleBatteryBAT1 = GObject.registerClass({
     }
 
     isAvailable() {
+        // check if framework_tool is installed and check if is framework laptop
+        if (fileExists("/usr/bin/framework_tool") && readFile('/sys/devices/virtual/dmi/id/sys_vendor').includes("Framework"))
```

Comment:
The `framework_tool` is not always in `/usr/bin`. Is there no way to check it's somewhere in the `$PATH`?

Reply:
Yes there is. Check the Dell.js. `GLib.find_program_in_path('smbios-battery-ctl') ? true : false;`
Anyways I have already created a patch for framework. But I can't test it. It gives an option to choose between sysfs and framework tool. I will submit the patch tomorrow as I didn't have time. It is not perfect and may require some work. You can test and correct what is needed.

Reply:
Thanks for pointing out the "/usr/bin" situation. I feel like having an option and letting the user choose isn't necessary. Nevertheless, if you already implemented it and only need testing it might be good closing this pull request?